cfpb/cfgov-refresh
# Updating node modules to latest versions where possible

Updating node modules to latest versions where possible


## Additions

- Updated the node nodules to the latest version where applicable.


## Changes

- Modified `package.json` to the dependencies to the latest version where possible.

## Testing

- Run `rm -rf node_modules`.
- Run `frontend.sh`
- Browse the site as expected.
- Run `gulp test:unit`


## Notes

- This PR didn't really resolve many of the High level vulnerabilities. This issue is that the dependencies  are deeply nested and pinned. Many of the dependency repos just aren't well maintained. 


## Checklist
